libkbfs/key_manager_test.go

```diff
@@ -2019,7 +2019,7 @@ func testKeyManagerRekeyAddDeviceWithPromptViaFolderAccess(t *testing.T, ver Met
 		defer ops.mdWriterLock.Unlock(lState)
 
 		// Now cause a paper prompt unlock via a folder access
-		errCh := make(chan error)
+		errCh := make(chan error, 1)
```

Comment:
> Ah, it's actually not this which I think may fix the hang, but the conversion to errGroup.

Code looks fine, though I still don't understand why the `errGroup` conversion could fix the hang.  In the current `processRange` code, there's always a default case when sending on `errChan`, and the stack traces in KBFS-1641 show the hang is in the decrypt call, not in anything that's changed by the conversion to `errGroup` as far as I can see.  What's your theory for why it fixes the hang?